packages/main/src/Calendar.js

```diff
@@ -143,52 +165,69 @@ const metadata = {
 /**
  * @class
  *
+ * <h3 class="comment-api-title">Overview</h3>
+ *
+ * The <code>ui5-calendar</code> can be used stand alone to display the years, months, weeks and days
+ * but the main purpose of the <code>ui5-calendar</code> is to be used within a <code>ui5-date-picker</code>
```

Comment:
remove the part "but the main purpose of the ui5-calendar is to be used within a ui5-date-picker"

Reply:
Done